Describe VectorMap - Custom Annotations (#247)
DevExpress/devextreme-demos: JSDemos/Demos/VectorMap/CustomAnnotations/description.md

> Annotations are images and text blocks that provide additional information about the visualized data.

Reviewer: ```suggestion
Annotations are images and text blocks that display additional information about the visualized data.
```

> Annotations are containers for images, text blocks and custom content that display additional information about the visualized data.

Reviewer 2: ```suggestion
Annotations are containers for images, text blocks, and custom content that display additional information about the visualized data.
```

> To configure individual annotations, assign an array of objects to the [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/) option. To specify settings for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over common settings.

Reviewer: The first sentence first talks about "individual" annotations, then switches to an array (so, multiple objects). That's a bit confusing... 

```suggestion
To add annotations to a chart, assign an array of objects to the [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/) option. To specify settings for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over common settings.
```

Author: I applied your suggestions, but let me clarify what I meant.

Each object in the annotations[] array represents an individual annotations, so users have to specif individual settings in these objects.

> Each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/#type) attribute should be set to *"text"*, *"image"*, or *"custom"*. In this demo, the type of annotations is *"custom"*.

Reviewer 2: ```suggestion
Each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/#type) attribute should be set to *"text"*, *"image"*, or *"custom"*. In this demo, the type of all annotations is *"custom"*.
```

> Each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/#type) attribute should be set to *"text"*, *"image"*, or *"custom"*. In this demo, the type of all annotations is *"custom"*.

Reviewer: Same comment as above - I don't like the word "should". It implies that I'm given an option to not set it at all. But then what happens?

> You can set each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/#type) attribute to *"text"*, *"image"*, or *"custom"*. In this demo, the type of all annotations is *"custom"*.

Reviewer 2: ```suggestion
You can set each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/#type) option to *"text"*, *"image"*, or *"custom"*. In this demo, the type of all annotations is *"custom"*.
```

> To create an annotation of this type, declare the SVG markup in the [template](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/#template) option. Inside the template, you can access the annotation being customized.

Reviewer: To create an annotation of that type, you set its type to "custom" :-) In this case we already talked about type, so now we need to talk about specifying content for these custom annotations.  

```suggestion
Custom annotations require that you specify your own display [template](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/#template) in SVG format. As you can see in demo code, you can access annotation data within the template markup.
```

> For more information on annotation settings, refer to the [annotations[]](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/) documentation section.

Reviewer: ```suggestion
For more information on annotation settings, refer to the [annotations[]](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/) help topic.
```
